pkg/types/types.go

```diff
@@ -69,6 +70,14 @@ type RuntimeConfig struct {
 	CNIDeviceInfoFile string          `json:"CNIDeviceInfoFile,omitempty"`
 }
 
+// LogOptions specifies the configuration of the log
+type LogOptions struct {
+	MaxAge     int  `json:"maxAge,omitempty"`
+	MaxSize    int  `json:"maxSize,omitempty"`
+	MaxBackups int  `json:"maxBackups,omitempty"`
+	Compress   bool `json:"compress,omitempty"`
+}
```

Comment:
In this structure, you defined each value as 'omitempty', so if user does not specify any of them, then 0 or false will be set to the variable, as https://pkg.go.dev/encoding/json#Marshal .

This means if user uses current multus config (which does not have any LogOption) everything is set to 0 or false. This is not what current user want to (at least, MaxAge and MaxSize are 0, hence current log is gone).

Could you think about the situation and provide how to solve that?